### PR TITLE
#143 EA-1764 Implemented bypass API key validation when service is down

### PR DESCRIPTION
**Pull request by rgraf**
Combined with commons-api.

### CODE REVIEW

File: entity-web/src/main/resources/entity-web-beans.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
    	<beans:bean name="entity_authenticationService" class="eu.europeana.entity.web.service.authentication.mock.BridgeAuthenticationManagerAdapter"/>
 	<beans:bean name="entity_authorizationService" class="eu.europeana.entity.web.service.authorization.AuthorizationServiceImpl"/> 	 
 	<beans:bean name="commons_oauth2_europeanaClientDetailsService" class="eu.europeana.api.commons.oauth2.service.impl.EuropeanaClientDetailsService">
-		<beans:property name="apiKeyServiceUrl" value="#{entityProperties['europeana.apikey.serviceurl']}"/>
+		<beans:property name="apiKeyServiceUrl" value="http://apikey2-test.eanadev.org/apikey"/>
```
Comment (gsergiu) [inline entity-web/src/main/resources/entity-web-beans.xml]: This is wrong, the code merges must not override updated funcitonality that is already committed to develop branch

Comment (gsergiu) [reply entity-web/src/main/resources/entity-web-beans.xml]: please revert the previous code

File: entity-web/src/main/java/eu/europeana/entity/web/service/authorization/AuthorizationServiceImpl.java [hunk 2]
```diff
@@ -148,4 +148,5 @@ protected Role getRoleByName(String name) {
     protected String getApiName() {
 	return getConfiguration().getAuthorizationApiName();
     }
+
```
Comment (gsergiu) [inline entity-web/src/main/java/eu/europeana/entity/web/service/authorization/AuthorizationServiceImpl.java]: why is this change commited to github?

File: entity-web/src/main/resources/entity-web-beans.xml (hunk 1 above)
Comment (rgraf) [reply entity-web/src/main/resources/entity-web-beans.xml]: Reverted

File: entity-web/src/main/java/eu/europeana/entity/web/service/authorization/AuthorizationServiceImpl.java (hunk 2 above)
Comment (gsergiu) [reply entity-web/src/main/java/eu/europeana/entity/web/service/authorization/AuthorizationServiceImpl.java]: @rgraf  well .. there was no change in the code, only one space added

**Pull request merged**